src/target/riscv/riscv.c

```diff
@@ -3296,6 +3308,7 @@ struct target_type riscv_target = {
 	.mmu = riscv_mmu,
 	.virt2phys = riscv_virt2phys,
 
+	.get_gdb_arch = riscv_get_gdb_arch,
```

Comment:
Off-hand do we know what user-visible effects this addition has?

Reply:
It might allow people to connect to a RISC-V target from gdb without first telling gdb which binary is being debugged, or whether it's riscv32/riscv64. It's been on my TODO list forever, so I'm glad somebody did it without me having to do anything. (You can read the upstream PR at http://openocd.zylin.com/#/c/6322/)